CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/sna/centrality/BetweennessCentralityPlugin.java

```diff
@@ -153,23 +153,31 @@ public void edit(final GraphWriteMethods graph, final PluginInteraction interact
         // update the graph with betweenness values
         for (int vertexPosition = 0; vertexPosition < vertexCount; vertexPosition++) {
             final int vertexId = graph.getVertex(vertexPosition);
+            float betweennessAttributeValue;
             if (normaliseByPossible) {
                 if (normaliseConnectedComponents) {
                     final float subgraphVertexCount = subgraphs[vertexPosition].cardinality();
+                    betweennessAttributeValue = betweennesses[vertexPosition] / (((subgraphVertexCount - 1) * (subgraphVertexCount - 2)) / 2);
                     graph.setFloatValue(betweennessAttribute, vertexId, betweennesses[vertexPosition] / (((subgraphVertexCount - 1) * (subgraphVertexCount - 2)) / 2));
```

Comment:
remove all these `graph.setFloatValue` statements as your new one replaces them.

Reply:
Whoops, forgot to remove them...my bad, that's quite a glaring omission.